js/page-specific/phone-directory/phone-directory.js

```diff
@@ -0,0 +1,131 @@
+var jQuery = $.noConflict(true);
+var pageCount = 0;
+
+function __onPrevPage() {
+    for (var i = 1; i <= pageCount; i++) {
+        if (document.getElementById("Page" + i).style.display == "block") {
+            document.getElementById("Page" + i).style.display = "none";
+            document.getElementById("Page" + (i - 1)).style.display = "block";
+            break;
+        }
+    }
+    setNextPrevious(i - 1);
+}
+
+function setNextPrevious(pageNum) {
+    if (pageNum < pageCount) {
+        document.getElementById("Next").style.display = "block";
+    } else {
+        document.getElementById("Next").style.display = "None";
+    }
+
+    if (pageNum > 1) {
+        document.getElementById("Previous").style.display = "block";
+    } else {
+        document.getElementById("Previous").style.display = "None";
+    }
+}
+
+function validateForm() {
+    var firstName = jQuery.trim(
```

Comment:
not blocking but you could so something like

```js
var firstName = $('#txtFirst').text().trim();
```

Reply:
Thanks for suggestion, updated this